HACKING.md

```diff
@@ -31,6 +58,9 @@ public Internet if desired.
 The second command will attempt to deploy directly to the namespace blog.
 Obviously you will need write access to the repository to accomplish this.
 
-Talk to @msanatan if you need to have something published. He is the current maintainer.
-
+As an example, let's say I just made a pull request to merge the branch `working-hard` from `metasansana/namespaceappclub.github.io`. The steps the maintainer would take are:
 
+1. Review that pull request for nonesense.
+2. If its ok then merge it into *his/her* working branch locally.
```

Comment:
Need some clarification in point 2. If you are the maintainer, would you use the namespaceappclub repo instead of a metasana fork?